understanding/21/label-in-name.html

```diff
@@ -9,23 +9,61 @@
+<figure>
+	<img alt="Icons for transforming text, including heading, bold, italic, quote, code and link, along with icons for ordered and unordered lists and other controls" src="img/rich-text-editor-detail.png">
+	<figcaption>Figure 1. A detail of the rich text editor in Github, showing a variety of unlabeled icons, including icons resembling text characters.</figcaption>
+</figure>
+<p>Likewise, where an author has used a greater-than symbol (">") to mimic the appearance of the right-facing arrow, the text does not convey something in human language. It is a symbol, in this scenario likely meant to mimic the icons used for a "Play" button or a "Next" arrow.</p>
+<h4>Punctuation and capitalization</h4>
+<p>The use of punctuation and capitalization in labels <em>may</em> also be considered optional for the same reason. For example, the colon conventionally added at the end of input labels does not express something in human language, and capitals on the first letter of each word in a label do not normally alter the words' meaning. This is particularly relevant in the context of this SC, since it is primarily aimed at users of speech recognition; capitals and most punctuation are frequently ignored when a user speaks a label as a means of interacting with a control.</p>
+<p>While it is certainly not an error to include the colon and capitalization in the accessible name, a computed name of "First name" should not be considered a failure of "First Name:".<br />
+<label for="firstname">First Name: </label><input id="firstname" type="text" name="firstname"><br />
+Likewise, "Next…" visibly shown on a button could have "Next" as the accessible name. When in doubt, where a meaningful visible label exists, match the string exactly for the accessible name.<br />
+<input type="submit" value="Next..."></p>
+
+<h4>Mathematical expressions and formulae</h4>
+<p>Mathematical expressions can be used as labels, for example "11x3=33" and "A>B" convey meaning. The label should not be overwritten in the accessible name, and substitutions of words where a formula is used should be avoided, such as making the name "eleven times three equals thirty-three" or "A is greater than B". Doing so would mean that a user who said "eleven multiplied by 3 is equivalent to thirty-three" would be unlikely to match. Further, converting a mathematical formula label into an accessible name that is a spelled-out equivalent may create issues for translation. The name should match the label's formula text.<br />
```

Comment:
Just a note to remind that this was the section Bruce had concerns about and needed a sentence.

Reply:
Bruce has signed off on my changes.